app/uk/gov/hmrc/vatsubscription/models/updateVatSubscription/request/DeregInfo.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.vatsubscription.models.updateVatSubscription.request
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{JsPath, Writes}
+
+case class DeregInfo(deregReason: Option[String],
+                     deregDate: Option[String],
+                     deregLaterDate: Option[String])
+
+object DeregInfo {
+
+  implicit val writes: Writes[DeregInfo] = (
+    (JsPath \ "deregReason").writeNullable[String] and
```

Comment:
I assume this doesn't have to be an option as we will always be supplying a reason when the deregistationInfo object is present?